Documentation/Signoffs/Sensors.md

```diff
@@ -0,0 +1,157 @@
+
+  
+
+ 
+
+## Constraints  
+
+  
+
+**The Sensor System:** 
+
+   
+
+* **Cannot take up more than three I/O pins from the PLC**    
+
+   ^The PLC module will have a limited amount of inputs and outputs to receive or send signals. All other pins will need to be allocated to receive from and or send signals to the locks, LEDs, and motor.  
+
+   ^This also makes the constraint that the sensors must send one bit signals since there will have to be three separate—one for each dial layer (which each take up one I/O pin). 
+
+ 
+
+* **The distance sensor chosen must determine the presence or absence of a device by measuring whether the light emitted was reflected back from a distance shorter than the distance between the sensor and the back wall of the compartment.** 
+
+   ^To understand why the sensor must be a light emitting distance sensor see the analysis section below. 
+
+   ^The sensor must not communicate the presence of a device by sensing the back wall of the compartment. 
+
+ 
+
+* **Must send a one signal to the PLC if there is a device in the presently forefront compartment section and a zero signal if there is not.**  
+
+   ^This constraint is chosen when considering the consequences in the case of the wire being cut: 
+
+If the default (zero) signal communicates that a device is present then the door could be able to be opened, upon someone trying to check a device out, because         the PLC thinks there is a device there to be distributed. 
+
+If the default (zero) signal communicates that a device is not present then the door will remain locked and cannot be unlocked because the PLC thinks there is no       device there to distribute. 
+
+Upon reflection the second option is the best because the devices are protected behind the locked door which will prevent theft during the time period before the       wire connection is fixed. 
+
+ 
+
+* **The sensor should be small enough to fit inside the palm of your hand so as not to obstruct access to the compartment** 
+
+   ^This should be sufficiently small and practically achievable to find a sensor of such size. 
+ 
+
+  
+
+## Schematic  
+
+![SpotSensorsSchematic1](https://user-images.githubusercontent.com/113734069/215000045-7e4d8fe0-7820-4f14-8b34-5ba22decf83a.jpg)
+
+
+We have designated three I/O pins—7, 8, and 9—of our PLC module to input signals from our sensors. Each of the three sensors has three wires; the signal wires (SDA) will be connected to their respective input pin of the PLC; the wires powering the sensors (AVDD) will be connected to terminal blocks--jumped together--to connect directly to the positive voltage wire of the power supply; the neutral wires of the sensors (NTRL) will also be connected to terminal blocks that are jumped together. As depicted in the diagram the top level, middle level, and bottom level sensor will be connected to pin 9, pin 8, and pin 7 of the PLC respectively. 
+
+  
+
+  
+
```

Comment:
Why is there irrelevant analysis here? 

Analyze your system and prove that it should work.